trains-server-chart/templates/trains-agent-deployment.yaml

```diff
@@ -42,8 +42,10 @@ spec:
             nvidia.com/gpu:
               {{ .Values.agent.nvidiaGpusPerAgent }}
         volumeMounts:
+          {{if not .Values.agent.dockerMode}}
```

Comment:
@valeriano-manassero shouldn't this be an `if` instead of `if not`?
If dockerMode is used then we do need the mount.

Reply:
You are absolutely right, sorry for this mistake. Fixed.